src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,40 @@
+import cn from 'classnames';
+import React from 'react';
+
+type Props = {
+  errorMessage: string,
+  setErrorMessage: (value: string) => void,
+  onCloseErrorMessage: () => void;
```

Comment:
Figure out how to pass just one function `closeError`
```suggestion
  closeError: () => void;
```